tuxemon/core/states/combat/combat.py

```diff
@@ -304,8 +313,10 @@ def transition_phase(self, phase):
             # TODO: proper match check, etc
             # This assumes that player[0] is the human playing in single player
             if self.remaining_players[0] == self.players[0]:
+                self.players[0].game_variables['battle_last_won'] = 'true'
```

Comment:
what happens if you run from the battle?  this value should be cleared at the beginning of a battle.

Reply:
I mainly went with that name to mirror the existing name in the locale string translation.

https://github.com/Tuxemon/Tuxemon/blob/development/tuxemon/core/locale.py#L304

Reply:
fair enough.  i think you meant to reply to my other comment?

Reply:
I didn't think of that. The values are replaced and recorded when any battle starts, so it's essentially the name of the last monster you met.

What if instead I changed the battle_last_won variable to have 3 values: Won, lost, ran? That way it will still record the last monster you encountered, but at the same time what happened when you met them... an NPC can then poke fun of you for seeing you run away from said monster.

Reply:
Yes, sorry about that: I moved my comment above on its line.

Reply:
"last_battle_result" seems fine to me.  possible values are lost, ran, won.  are there draw matches?

Reply:
very minor, please rename "last_battle_status" to "last_battle_result"

Reply:
There is a draw match phase. It might be used or could be in the future, so it's not a bad idea to add a scenario for that as well.

Reply:
Renamed it to battle_last_result. I'd rather keep the battle_ prefix, as such variables might be used for other things in the future and this keeps them cleanly organized.